pkgs/applications/version-management/gitlab/default.nix

```diff
@@ -89,7 +89,7 @@ let
 
       bundle exec rake gettext:po_to_json RAILS_ENV=production NODE_ENV=production
       bundle exec rake rake:assets:precompile RAILS_ENV=production NODE_ENV=production
-      bundle exec rake webpack:compile RAILS_ENV=production NODE_ENV=production NODE_OPTIONS="--max_old_space_size=2048"
+      bundle exec rake webpack:compile RAILS_ENV=production NODE_ENV=production NODE_OPTIONS="--max_old_space_size=4096"
```

Comment:
Well, this used to be set to `4096`, but that fails when building on i686. Can you try a slightly smaller increase?

Reply:
Yes, I will now try 3072 and if it works I'll commit that.

Reply:
Works!